.fttemplates/TS Functional Component/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { <FTName | capitalize> } from './[FTName]'
+
+export { <FTName | capitalize> }
```

Comment:
like this is enough since it is not being used inside the file
```suggestion
export { <FTName | capitalize> } from './[FTName]'
```